.github/workflows/build.yml

```diff
@@ -74,6 +72,12 @@ jobs:
       run: |
         echo "GHC_VERSION=$GHC_VER" >> $GITHUB_ENV
 
+    - name: Tentative Workaround for GHC 8.10.5 on macOS
+      if: matrix.os == 'macOS-latest' && matrix.ghc == '8.10.5'
+      run: |
+        echo "# uninstalling CommandLineTools (see https://github.com/haskell/haskell-language-server/issues/1913#issuecomment-861667786)"
+        sudo rm -rf /Library/Developer/CommandLineTools
+
```

Comment:
Could you also apply the similar trick to test CI so that we can test 8.10.5 build on macOS for each PR?

Reply:
Good point! Added to test.yml.